guides/common/modules/proc_migrating-project-or-smartproxy-using-clone.adoc

```diff
@@ -0,0 +1,22 @@
+[id="migrating-{project-context}-or-proxy-using-clone_{context}"]
+= Migrating {Project} or {SmartProxy} to {EL-abbr}{nbsp}9 using clone
+
+You can clone your existing {ProjectServer} and {SmartProxyServer} from {EL}{nbsp}8 to a freshly installed {EL}{nbsp}9 system.
+ifdef::satellite[]
+Create a backup of the existing {ProjectServer} and {SmartProxyServer}, which you then clone on the new {EL}{nbsp}9 system.
+[NOTE]
+====
+You can use cloning for {ProjectServer} backups but not for {SmartProxyServer} backups.
+====
+endif::[]
+
+.Procedure
+. Perform a full backup of your {ProjectServer} or {SmartProxy}.
+This is the source {EL}{nbsp}8 server that you are migrating.
+For more information, see {AdministeringDocURL}backing-up-{project-context}-server-and-{smart-proxy-context}_admin#Performing_a_Full_Backup_admin[Performing a full backup of {ProjectServer} or {SmartProxyServer}] in _{AdministeringDocTitle}_.
+. Deploy a system with {EL}{nbsp}9 and the same hostname and configuration as the source server.
+This is the target server.
+ifdef::satellite[]
+. Clone the server.
+Clone configures hostname for the target server and you can test it before retiring the source server.
+For more information, see {AdministeringDocURL}sec-Cloning_to_Target[Cloning to the target server] in _{AdministeringDocTitle}_endif::[]
```

Comment:
Note the missing new line, it breaks the link and the conditional.

Reply:
Also, how about making the link point to the assembly rather than a procedure? I think that would work better because the assembly includes a section on how to prepare the server for cloning, you probably don't want users to skip it.